var/spack/repos/builtin/packages/py-htseq/package.py

```diff
@@ -10,11 +10,20 @@ class PyHtseq(PythonPackage):
     """HTSeq is a Python package that provides infrastructure to process
     data from high-throughput sequencing assays."""
 
-    homepage = "https://htseq.readthedocs.io/en/release_0.9.1/overview.html"
-    url = "https://github.com/simon-anders/htseq/archive/release_0.9.1.tar.gz"
+    homepage = "https://htseq.readthedocs.io/en/master/index.html"
+    url = "https://github.com/htseq/htseq/archive/release_0.12.3.tar.gz"
```

Comment:
Can we instead download from PyPI? Then the URL doesn't have to change. See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Reply:
I've switched everything over to PyPI and regenerated the checksums in a new commit.